crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -917,6 +921,14 @@ impl AuthorityPerEpochStore {
                 None
             };
 
+        let consensus_tx_status_cache = if protocol_config.mysticeti_fastpath() {
+            Some(ConsensusTxStatusCache::new(
+                protocol_config.gc_depth() as u64
```

Comment:
I think this can be a constant or per-validator config later?
gc_depth is only 4-5 sec worth of transactions which seem to be too few. I think keeping 30-60 sec of past transaction statuses should be cheap enough.
Ok to leave a todo.